server/pkg/mail/healthcheck.go

```diff
@@ -17,3 +25,39 @@ func Healthcheck(ctx context.Context) error {
 
 	return nil
 }
+
+func IsValidHost(host string) bool {
```

Comment:
are these new functions here really healtchecking functions or just... checking functions? I would rather see them in a different file